components/brave_shields/browser/brave_resource_dispatcher_host_delegate.cc

```diff
@@ -30,6 +38,31 @@ void BraveResourceDispatcherHostDelegate::AppendStandardResourceThrottles(
     content::ResourceContext* resource_context,
     ResourceType resource_type,
     std::vector<std::unique_ptr<content::ResourceThrottle>>* throttles) {
+
+  CHECK(g_brave_browser_process->tracking_protection_service()->IsInitialized());
+  ProfileIOData* io_data = ProfileIOData::FromResourceContext(resource_context);
+  HostContentSettingsMap* contentSettingsMap = io_data->GetHostContentSettingsMap();
+
+  if (!g_brave_browser_process->tracking_protection_service()->addFirstPartyTrackers().empty()) {
+    for(std::string tracker : g_brave_browser_process->tracking_protection_service()->addFirstPartyTrackers()) {
```

Comment:
I don't think these should go into user content settings. That will potentially change or wipe out user specified settings

Reply:
these need to come from a separate content settings provider as we discussed in slack